src/client/pages/OfferNew/Introduction/Sidebar/DetailsModal/Details/NorwegianDetails.tsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react'
+import { FormikProps } from 'formik'
+import { InputGroup } from 'components/inputs'
+import { EditQuoteInput } from 'data/graphql'
+import { TextKeyMap } from 'src/client/utils/textKeys'
+import {
+  isNorwegianHomeContentFieldSchema,
+  isNorwegianTravelFieldSchema,
+} from '../utils'
+import {
+  NorwegianTravelContentFieldSchema,
+  NorwegianHomeContentFieldSchema,
+} from '../types'
+import { DetailInput } from './components/DetailInput'
+import { SupportSection } from './components/SupportSection'
+import {
+  Content,
+  ContentColumnTitle,
+  ContentColumn,
+} from './components/Details.styles'
+import { DetailsProps } from './types'
+
+export const NorwegianDetails = ({
+  fieldSchema,
+  formikProps,
+  offerQuote,
+  textKeys,
+}: DetailsProps) => (
```

Comment:
I actually disagree about it being more clear - I think it's harder to read and looks... messier. And also my understanding is that both ways are nowadays equally correct, for example according to [this cheatsheet](https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/function_components) that is linked to from the TypeScript docs.

But with that said - since we do it the `export const MyComponent: React.FC<SomeProps>` way in all other files I agree about changing it here! I think it's way more important that we are consistent 😇

Reply:
Interesting and good read! I guess the easiness-to-read and messiness level of it is about habit because I do feel the same about writing the prop type as the prop argument haha 😅